packages/fiori-elements-writer/src/types.ts

```diff
@@ -31,12 +31,15 @@ export interface EntityConfig {
         EntitySet: string; // Defines the entity set for object page navigation
         Name: string; // Defines the entity name for object page navigation
     };
+    tableType?: TableType;
```

Comment:
Would it be better to add these to the settings type for each supporting template. IM not sure they belong in `EntityConfig` ?

Reply:
Addressed in https://github.com/SAP/open-ux-tools/pull/1833/commits/6afb1265e55563eb8ca01296620f6baa02f7a11d